### PR TITLE
Addition of Schema Registry Operator

### PR DESCRIPTION
**Pull request by vibhujain**
Confluent Schema Registry provides a serving layer for your metadata.
Solves #149 
Signed-off-by: Vibhu Jain <vibhujain.j@gmail.com>

### CODE REVIEW

File: repository/schema-registry/operator/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Name }}
+  namespace: {{ .Namespace }}
+  labels:
+    app: schema-registry
+spec:
+  replicas: 1
```
Comment (zmalik) [inline repository/schema-registry/operator/templates/deployment.yaml]: let's parametrize this

File: repository/schema-registry/operator/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: schema-registry
+    spec:
+      containers:
+      - name: schema-registry
+        image: mesosphere/cp-schema-registry:5.3.0
```
Comment (zmalik) [inline repository/schema-registry/operator/templates/deployment.yaml]: and use the confluent image

**Review by zmalik — CHANGES_REQUESTED** [review]
this is looking good! 
leaving just two comments

File: repository/schema-registry/operator/templates/deployment.yaml (hunk 2 above)
Comment (vibhujain) [reply repository/schema-registry/operator/templates/deployment.yaml]: Done. Changed the image to Confluentic one.

Comment (zmalik) [reply repository/schema-registry/operator/templates/deployment.yaml]: and let's use the version `5.3.2`

File: repository/schema-registry/operator/templates/deployment.yaml (hunk 1 above)
Comment (vibhujain) [reply repository/schema-registry/operator/templates/deployment.yaml]: Done. Added parameters to configure this.

File: repository/schema-registry/operator/templates/deployment.yaml (hunk 2 above)
Comment (vibhujain) [reply repository/schema-registry/operator/templates/deployment.yaml]: Done.

File: repository/schema-registry/operator/operator.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+apiVersion: kudo.dev/v1beta1
+name: "schema-registry"
+version: "0.1.0"
+kudoVersion: 0.8.0
+kubernetesVersion: 1.15.0
+appVersion: 5.3.0
```
Comment (zmalik) [inline repository/schema-registry/operator/operator.yaml]: app version is `5.3.2`

**Review by zmalik — CHANGES_REQUESTED** [review]

File: repository/schema-registry/operator/operator.yaml (hunk 3 above)
Comment (vibhujain) [reply repository/schema-registry/operator/operator.yaml]: Done. Thanks for the correction!

**Review by zmalik — APPROVED** [review]
LGTM! 
thanks @vibhujain

**Comment by guenter** [comment]
This operator should be called "confluent-schema-registry" so it's clear that it is for Confluent/Kafka. That's also the official name, see https://github.com/confluentinc/schema-registry

**Comment by zmalik** [comment]
opened a PR to fix that https://github.com/kudobuilder/operators/pull/175
I will fix the KUDO repository once the https://github.com/kudobuilder/operators/pull/175 is merged

**Pull request merged**